molecule/provisioner/ansible/playbooks/lxd/prepare.yml

```diff
@@ -3,51 +3,30 @@
   hosts: all
   gather_facts: false
   tasks:
-    - name: Test if Python is installed
```

Comment:
Simply skip this check and just always confirm python and other basic packages are installed correctly.